modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -158,6 +158,36 @@ module UpdatePoaRequest
         end
       end
 
+      ##
+      # PersonWebServiceBean
+      #
+      module PersonWebServiceBean
+        DEFINITION =
+          Bean.new(
+            path: 'PersonWebServiceBean',
+            namespaces: Namespaces.new(
+              target: 'http://person.services.vetsnet.vba.va.gov/',
+              data: 'http://person.services.vetsnet.vba.va.gov/'
+            )
+          )
+      end
+
+      module PersonWebService
+        DEFINITION =
+          Service.new(
+            bean: PersonWebServiceBean::DEFINITION,
+            path: 'PersonWebService'
+          )
+        module FindPersonBySSN
+          DEFINITION =
+            Action.new(
+              service: PersonWebService::DEFINITION,
+              name: 'findPersonBySSN',
+              key: 'PersonDTO'
+            )
+        end
+      end
```

Comment:
Are we wanting to skip WSDL fetching for this action as soon as this code lands?

Reply:
Also a nit pick, but if we do want this, maybe add an empty line above line 181.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/17185/files#r1648017735 - Yeah, like this on a Flipper Feature.